utils_test.go

```diff
@@ -438,22 +438,25 @@ func TestFileExists(t *testing.T) {
 }
 
 func TestGetWithin(t *testing.T) {
+	Now = func() time.Time { return time.Date(2019, time.December, 25, 0, 0, 0, 0, time.UTC) }
 	cs := []struct {
```

Comment:
I believe that assign this function to `Now` would be misleading, because may give people the impression that this refers to the current time, while it's not.

@slaskawi what do you think about these changes?